File: python/Program/produk_elektronik.py
from produk import Produk

# produkElektronik turunan dari produk
class ProdukElektronik(Produk):
    # Constructor
    def __init__(self):
        super().__init__()
        self.__merk = ""
        self.__garansi = 0  # bulan
        self.__model = "" 

    # Getter
    def get_merk(self):
        return self.__merk

    def get_garansi(self):
        return self.__garansi
    
    def get_model(self):
        return self.__model
    
    # Setter
    def set_merk(self, merk):
        self.__merk = merk

    def set_garansi(self, garansi):
        self.__garansi = garansi

    def set_model(self, model):
        self.__model = model

File: php/Program/class/ProdukElektronik.php
<?php
require_once "Produk.php";

// produkElektronik turunan dari produk
class ProdukElektronik extends Produk {
    // atribut class ProdukElektronik
    private $merk;
    private $garansi; // dalam bulan
    private $model; // model laptop

    // Constructor
    public function __construct($id = "", $nama = "", $harga = 0, $stok = 0, $gambar = "",
                                $merk = "", $garansi = 0, $model = "") {
        parent::__construct($id, $nama, $harga, $stok, $gambar);
        $this->merk = $merk;
        $this->garansi = $garansi;
        $this->model = $model;
    }

    // Getter
    public function getMerk() { return $this->merk; }
    public function getGaransi() { return $this->garansi; }
    public function getModel() { return $this->model; }

    // Setter
    public function setMerk($merk) { $this->merk = $merk; }
    public function setGaransi($garansi) { $this->garansi = $garansi; }
    public function setModel($model) { $this->model = $model; }
}
